src/pdf_parser/pdf_utils/disambiguate_layout.py

```diff
@@ -155,7 +155,7 @@ def reduce_overlapping_boxes(
 
     if direction == "vertical":
         assert (
-            box_1.coordinates[1] < box_2.coordinates[1]
+            box_1.coordinates[3] < box_2.coordinates[3]
```

Comment:
Is this correct? Comparing the x co-ordinate for vertical?

Reply:
It's not the x coordinate. The coordinates are (x1,y1,x2,y2), where (x1, y1) is top left and (x2,y2) is bottom right. I altered this for consistency with the call of the parent function, so that both use y2 coords instead of one using y1 and the other y2 (it doesn't actually matter except when we hit the edge case discussed earlier).

Reply:
Ahh yeah my bad 0,1,2,3 x,y,x,y